client/src/pages/ManageBookings/mockBookingData.ts

```diff
@@ -0,0 +1,51 @@
+import { BookingStatus } from './types';
+
+const mockBookingData = [
+  {
+    description: 'Test Booking Request Description #1',
+    status: BookingStatus.pending,
+    requestId: '61a26c5dc517da0ca7a2b9',
+    userId: '619c203917b5e66160b119fb',
+    sitterId: '618ff3d3939f8555dc391646',
+    start: new Date(2021, 10, 9),
+    end: new Date(2021, 10, 11),
+  },
```

Comment:
First of all, well done using mocks 🎉 - it's a best practice that's overlooked often! This is a very good start, but we can improve on it: the format of a mock data shouldn't necessarily be identical to the backend model of what we are showing (e.g. `Request` in this case). Instead, it should be tailored to what the frontend expects - and it's important to realize this can vary from how we store things in the backend.

Let's consider our use case: 
- We need the details of a booking to filter and display it properly, so it makes sense to expect fields such as `status`, `description`, `start` and `end`.
- The mocks show us the booking cards need to display the user's name and profile picture, hence we should expect those. To me it makes more sense to expect a field such as `ownerProfile` instead of user id. I can't really work with the user id - if I can't display a user's name or profile picture with it, what good is it to me? Right?
- Since this page is for sitters only, sitter id is going to be my (logged in user's) user id (or profile id depending on BE implementation). I already have access to both, do I need to expose it to the frontend? Are we going to do something with it? (no I'm asking seriously, I might be missing something :D) It doesn't seem like my mock data needs to include that at a first glance. Besides I'm always against exposing user ids unnecessarily, goes for both user and sitter ids - you can never be too safe 😉

Clearly and explicitly conveying exactly what you expect in your mock data makes integration soo much simpler 😮If I were to integrate this page and just retrieved the bookings from the API like the mocks, I would soon realize I need to do more work on the FE (such as retrieve the owner's user and it's profile, then integrate them in the appropriate parts which involve more FE work). If your mocks are formatted exactly as you expect the data to be passed from the "API layer", then my only work would be to write the appropriate fetching logic (i.e either populate on the backend (ideal) or do a few fetches to populate the necessary fields manually). 

In summary, it should be frontend's job to identify what it needs from the BE and it should be integration's job to figure out how to get it!

Hopefully I wasn't too confusing here😅 Feel free to ask for further clarification if needed 👍🏼

Reply:
@ugur-akin This (and your other comments) have been super helpful.  Thank you for taking the time for this level of explanation. I am time-limited at the moment and am not going to fully revise the mock, but certainly will utilize this feedback in the future.